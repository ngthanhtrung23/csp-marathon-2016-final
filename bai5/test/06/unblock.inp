..1233
..124.
00124.
.5..6.
.5776.
......
