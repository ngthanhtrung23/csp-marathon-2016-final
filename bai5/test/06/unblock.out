18
1 D 1
2 D 1
3 L 2
4 U 1
6 U 1
7 R 2
1 D 2
2 D 2
0 R 2
5 U 3
0 L 2
1 U 2
2 U 2
7 L 4
1 D 2
2 D 2
6 D 1
0 R 4

